doc/source/reference/random/bit_generators/index.rst

```diff
@@ -105,6 +105,35 @@ If you need to generate a good seed "offline", then ``SeedSequence().entropy``
 or using ``secrets.randbits(128)`` from the standard library are both
 convenient ways.
 
+If you need to run several stochastic simulations in parallel, best practice
+is to construct a random generator instance for each simulation. To make sure
+the random streams are independent, you can use the `spawn` method of
```

Comment:
independent is probably not the correct work. I think a better description of this would bt 

To make sure that the random streams have distinct initial states, ...